### PR TITLE
Add `empty-comment` example

### PR DESCRIPTION
**Pull request by harupy**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
| ✓   | :scroll: Docs          |

## Description

Refs #5953

Closes #XXXX

### CODE REVIEW

File: doc/test_messages_documentation.py [hunk 1]
```diff
@@ -121,6 +121,8 @@ def get_expected_messages(stream: TextIO) -> MessageCounter:
             line = match.group("line")
             if line is None:
                 lineno = i + 1
+            elif line.startswith("+") or line.startswith("-"):
+                lineno = i + 1 + int(line)
```
Comment (Pierre-Sassoulas) [inline doc/test_messages_documentation.py:125]: Thank you we needed that 👌 Good job on copying how it's done in functional tests too !

**Review by Pierre-Sassoulas — APPROVED** [review]
Perfect !

File: doc/test_messages_documentation.py (hunk 1 above)
Comment (Pierre-Sassoulas) [reply doc/test_messages_documentation.py:125]: I think the tailing newline example was broken, do you mind fixing it in this pull request directly ?

Comment (harupy) [reply doc/test_messages_documentation.py:125]: Fixed!

Comment (Pierre-Sassoulas) [reply doc/test_messages_documentation.py:125]: Thanks 👍

**Pull request merged**